pkg/feature/api/feature.go

```diff
@@ -455,11 +455,6 @@ func (s *FeatureService) listFeaturesFilteredByExperiment(
 			Operator: mysql.OperatorEqual,
 			Value:    false,
 		},
-		{
-			Column:   "experiment.deleted",
-			Operator: mysql.OperatorEqual,
-			Value:    false,
-		},
```

Comment:
Using `experiment.deleted` in the `ON` when joining a table and in the WHERE condition gives different results.
When the hasExperiment is set to false, it returns zero results, but it should return all flags not in experiments.